### PR TITLE
PLF-6993: Update README.txt, Release Notes and Changelog for PLF 4.4.0

### PR DESCRIPTION
**Pull request by mgreau**
This PR has been created to work on the Release Notes, Changelog and README files for PLF 4.4.0.

For now, I've done the following:

- [x] README.txt:  update HTTP link for eXo Platform documentation
- [x] CHANGE_LOG.txt : remove 4.3.x features,fix and improvement
- [x] RELEASE_NOTES : Prepare the `Main changes since Platform 4.3` part

The following parts have to be updated/created:

- CHANGE_LOG.txt
    - [x] New features
    - [x] Improvement
    - [x] Bug

- RELEASE_NOTES 
    - [x]  Main changes since Platform 4.3 > Features
    - [x]  Main changes since Platform 4.3 > Upgrade
    - [ ]  IV. KNOWN ISSUES

### CODE REVIEW

**Comment by thomasdelhomenie** [comment]
Changelogs have been updated with issues list and release notes have been updated with upgrade notes and architecture changes.
The only remaining item is Know Issues. I don't have in mind any new know issues to add in the list, but we should check if the already listed know issues are still issues in 4.4.0.

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 1]
```diff
@@ -3,318 +3,172 @@ Changelog Platform ${org.exoplatform.platform.version}
...
+    * PFR-1110 - @mentions in documents comments
+    * PFR-1111 - Align Document versioning screen with wiki
+    * PFR-1113 - Share Document to user
+
+** Tasks
+    * ECMS-7447 - Remove unused gadgets
+
+** Improvements
+    * ECMS-7430 - Stop calling /rest/state/ping if the last call failed
+    * ECMS-7431 - upgrade ITHIT to v2.0.1735.0
```
Comment (plamarque) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: [Open in Office] upgrade ITHit to 2.0.1735.0

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 2]
```diff
@@ -3,318 +3,172 @@ Changelog Platform ${org.exoplatform.platform.version}
...
+    * PFR-1117 - Add excerpt of the comment in notifications
+    * PFR-1131 - Text formatting for comments
+    * SOC-5487 - Disable Activities by type by configuration
+    * Social RDBMS in PLF
+
+** Tasks
+    * SOC-5211 - [Activity] Missing the translation of user's gender
+
+** Improvements
+    * SOC-5351 - Make people search tolerant to accents and typos
```
Comment (plamarque) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: accents and typos insensitive

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 3]
```diff
@@ -3,318 +3,172 @@ Changelog Platform ${org.exoplatform.platform.version}
...
 *************************
 
-** Bug
-    * [INTEG-417] - Avoid systemComment variable on postActivity of org.exoplatform.wcm.ext.component.activity.listener.Utils
-    * [INTEG-423] - [integ-ecms] Comment when updating summary for content is not right
-    * [INTEG-426] - wrong position of x icon
-    * [INTEG-432] - [Unified Search] Missing content type "Task" in the Filter
-    * [INTEG-435] - Bad display of Recent Activities in my profil
+** Improvements
+    * PFR-1116 - Forum Redesign
```
Comment (plamarque) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: overstatement. a more accurate summary would be : [Forum] Mobile friendly topic list

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt [hunk 4]
```diff
@@ -15,6 +15,70 @@ IV.   KNOWN ISSUES
 
 Main changes since Platform 4.3
 
+* Features
+** New file storage (File RDBMS)
```
Comment (plamarque) [inline plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: can we regroup technical features at the bottom of the list ?

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt [hunk 5]
```diff
@@ -75,9 +139,26 @@ However, if you still insist on skipping versions, we strongly advise to read al
 #PLF-6737: No signature of method: static org.exoplatform.services.deployment.Utils.sanitize() when using the Announcement template
 commons.upgrade.NodeTypeTemplateUpgradePlugin.enable=true
 
+In addition, upgrade scripts are automatically run at startup in this version to migrate wiki and social data from JCR to JPA.
+Wiki data migration is done synchronously, meaning that the eXo Platform instance will be available only once the data migration is done.
+Social data migration is done asynchronously, meaning that the data migration will run after the eXo Platform start.
+
+=== Architecture changes ===
+
+Since the version 4.4.0, eXo Platform needs Elasticsearch.
```
Comment (plamarque) [inline plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: /s/needs/requires

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt [hunk 6]
```diff
@@ -75,9 +139,26 @@ However, if you still insist on skipping versions, we strongly advise to read al
...
+Exo Platform is provided with an embedded version of Elasticsearch which automatically starts when eXo Platform starts.
```
Comment (plamarque) [inline plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: **e**Xo Platform is **shipped**

**Review by plamarque — CHANGES_REQUESTED** [review]
see my comments

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt (hunk 1 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: I used the jira issue summary as is so I will update jira issues as well.

Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: Done

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt (hunk 2 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: Done

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt (hunk 3 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: Done

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt (hunk 4 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: Done

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt (hunk 5 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: Done

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt (hunk 6 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: Done

**Pull request merged**